### PR TITLE
#1684: Disabling Latency statistics

### PR DESCRIPTION
**Pull request by mathieucarbou**

### CODE REVIEW

File: management/src/main/java/org/ehcache/management/providers/statistics/EhcacheStatisticsProvider.java [hunk 1]
```diff
@@ -55,6 +69,17 @@ protected void dispose(ExposedObject<CacheBinding> exposedObject) {
   }
 
   @Override
+  public final Collection<? extends Descriptor> getDescriptors() {
+    Collection<StatisticDescriptor> capabilities = new HashSet<StatisticDescriptor>();
+    for (ExposedObject o : getExposedObjects()) {
+      capabilities.addAll(((StandardEhcacheStatistics) o).getDescriptors());
+    }
+    List<StatisticDescriptor> list = new ArrayList<StatisticDescriptor>(capabilities);
+    Collections.sort(list, STATISTIC_DESCRIPTOR_COMPARATOR);
```
Comment (mathieucarbou) [inline management/src/main/java/org/ehcache/management/providers/statistics/EhcacheStatisticsProvider.java]: Note: I forgot to apply this code before. I also did this change for server entity stats. 
This really helps during tests or in TMS/TMC when generating a json view of the backend topology with management data. Having them unsorted makes it hard to find wrong stt items.

Comment (GaryWKeim) [inline management/src/main/java/org/ehcache/management/providers/statistics/EhcacheStatisticsProvider.java]: Well, you know in your heart that this should be in a different commit.

Comment (mathieucarbou) [inline management/src/main/java/org/ehcache/management/providers/statistics/EhcacheStatisticsProvider.java]: Yes, should have been! Now... Is it worth loosing another  1h for the new build to pass for this small thing and delaying more the PR for M&M ? :-S

Comment (gibsong) [inline management/src/main/java/org/ehcache/management/providers/statistics/EhcacheStatisticsProvider.java]: looks good to me, anything that helps with faster testing is always a plus.

**Comment by gibsong** [comment]
looks good to merge
